src/include/sof/low_latency_schedule.h

```diff
@@ -72,13 +72,14 @@ struct work_queue_timesource {
 };
 
 /* initialise our work */
-#define work_init(w, x, xd, xpriority, xflags) \
-	do { \
-		(w)->cb = x; \
-		(w)->cb_data = xd; \
-		(w)->priority = xpriority; \
-		(w)->flags = xflags; \
-	} while (0)
+static inline void work_init(struct work *w,
+	uint64_t(*x)(void *data, uint64_t udelay), void *xd,
+	uint32_t xpriority, uint32_t xflags) {
```

Comment:
Please name the function cb instead of x.

Reply:
It's done. Now cb and data exists only in generic sof_task struct